templates/app.html

```diff
@@ -40,7 +40,6 @@
   <!-- Styles -->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/css/materialize.min.css">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="static/css/dist/app.min.css" rel="stylesheet">
 
   <!-- <link rel="stylesheet" href="{{ url_for('static', filename='css/dist/bundle.css') }}"> -->
   <link rel="stylesheet" href="http://127.0.0.1:8080/static/css/dist/bundle.css">
```

Comment:
Below this, lines 48-50, I don't think we need to prefetch these gifs anymore. In fact, I think we can remove the gifs completely since we are using the svg file now.

```html
    <!-- Prefetch mic images -->
   <link rel="prefetch" href="/static/img/mic-slash.gif">
   <link rel="prefetch" href="/static/img/mic-animate.gif">
   <link rel="prefetch" href="/static/img/mic.gif">
```